sdk/search/search-documents/src/searchIndexerClient.ts

```diff
@@ -516,6 +516,8 @@ export class SearchIndexerClient {
           ...dataSourceConnection,
           credentials: {
             connectionString: dataSourceConnection.connectionString
+              ? dataSourceConnection.connectionString
+              : "<unchanged>"
```

Comment:
some nicer sugar for this:

```suggestion
            connectionString: dataSourceConnection.connectionString ?? "<unchanged>"
```